frontend/src/app/components/network-graph/network-graph.component.ts

```diff
@@ -509,15 +509,20 @@ export class NetworkGraphComponent implements OnInit, OnChanges {
     return boxY + (offset / 2);
   }
 
-
+  // noinspection JSMethodCanBeStatic
   private nodeInfo(d: Node): string[] {
-    let strings: string[];
-    strings = [
+    const strings: string[] = [];
+
+    if (d.id === '0x02f4b6BC65561A792836212Ebc54434Db0Ab759a') {
```

Comment:
Great idea, but can you add this to the config file. we need to be able to adjust this per network.

Reply:
It is part of the configuration file now :)